app/views/transactions/_transactions.html.slim

```diff
@@ -9,9 +9,10 @@
     tbody
     - @transactions.each do |record|
       tr
-        th= record.transactinable.date
+        th= record.transactinable.try(:date) || record.transactinable.created_at.strftime("%Y-%m-%d")
```

Comment:
@sinpai 
```ruby
record.transactinable.try(:date) || record.transactinable.created_at.strftime("%Y-%m-%d")
```

is used several times, the same as `"To #{record.transactinable.category.name} category"`


What do you think about moving that to decorator, see [gem](https://github.com/drapergem/draper)
You will have something like:
```ruby
# app/decorators/article_decorator.rb
class TransactionDecorator < Draper::Decorator
  delegate_all

  def date
    transactinable.try(:date) || transactinable.created_at.strftime("%Y-%m-%d")
  end

  ...
end
```